### DISCUSSION TITLE
[Opinion] Integrate Mesop into a Python backend web framework.

### DISCUSSION THREAD

**Discussion by rroblf01** [google/mesop#510] [Ideas]
It would be very interesting to be able to integrate Mesop into a backend project we have with a Python web framework such as Django/Fastapi/Flask.

**Comment by wwwillchen** [comment]
Right now Mesop is built on Flask, so it's possible that we could expose the Flask app. Can you share a little more about your use case? For example, do you need to serve HTTP endpoints in addition to a UI?

**Comment by rroblf01** [comment]
Good morning @wwwillchen 

Yes, at least for the use I can give it, it would be very useful to be able to make the backend in Django or FastApi and expose both the backend endpoints and a UI.
